source/nodejs/adaptivecards-designer/src/adaptivecards-designer.css

```diff
@@ -36,7 +36,8 @@
 }
 
 .acd-designer-host {
-    margin: 20px 40px 20px 20px;
+    margin: 20px 0;
+    padding: 0 40px 0 20px;
```

Comment:
How does this look when the entire Designer has a narrower width? 

![image](https://user-images.githubusercontent.com/16614499/155599943-57d2b008-9c94-4f23-8d11-7f6d9d01a0fe.png)

Reply:
This is how it looks in Edge, from a comfortable width down to when the viewport begins scaling (this behavior appears to be automatic since there is no viewport `<meta>`):

![pr](https://user-images.githubusercontent.com/855039/155673107-b6aeb517-e15e-40d5-824d-30629e9949bd.gif)

Reply:
You can emulate the change this PR introduces by duplicating the host app selector element as seen in production. Elements in the toolbar continue to wrap as before:

![pr](https://user-images.githubusercontent.com/855039/155798966-59a791ba-23ab-4925-a459-d53813e96dae.gif)

The toolbar pattern used in the designer conforms to WCAG SC 1.4.10 even with this PR, but the design lacks any appeal. IMO the designer should be redesigned responsively (i.e. mobile-first), but that would seem well out-of-scope for this PR.

Reply:
agree that the designer could definitely be more mobile-friendly... I retrofitted this reflow behavior to address accessibility concerns, but didn't have enough time to do a proper redesign. thanks for the gif! :)